src/transformers/models/bert_generation/modeling_bert_generation.py

```diff
@@ -785,9 +785,7 @@ def forward(
 
         # We can provide a self-attention mask of dimensions [batch_size, from_seq_length, to_seq_length]
         # ourselves in which case we just need to make it broadcastable to all heads.
-        extended_attention_mask = None
-        if not use_cache:
-            extended_attention_mask: torch.Tensor = self.get_extended_attention_mask(attention_mask, input_shape)
```

Comment:
when `use_cache` is specified and the model is used as decoder, this prevents the model to prepare the correct attention mask and each token attends to every other tokens.